gen_netdevs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+shopt -u expand_aliases
+ip a | grep 'inet ' | awk '{print $2}' | sed 's/\/.*$//g' > net_devices
+echo '0.0.0.0' >> net_devices
```

Comment:
Hm, that's odd. We just worked on this yesterday because I had a similar issue myself. Could you check what `vim net_devices` return? Does the first two entry have `^[[35m....^[[0m`, or anything other than the ip addresses?

Reply:
No, I only have ip addresses. But, before, I don't think that I even had the ip command. I think that that was the problem.

Reply:
We should probably just standardize this for Linux.